src/__tests__/shared/components/input/DatePickerWithLabelFormGroup.test.tsx

```diff
@@ -49,7 +49,7 @@ describe('date picker with label form group', () => {
         const datepickerInput = screen.getByRole('textbox')
 
         expect(datepickerInput).toHaveDisplayValue(['01/01/2019'])
-        userEvent.type(datepickerInput, '12/25/2021')
+        userEvent.type(datepickerInput, '{selectall}12/25/2021')
```

Comment:
Apologies if this is a silly question, but care to explain what is going on here? I saw something similar in other files above. I'm just not sure i get what is going on if you don't mind explaining.

Reply:
@blestab are you referring to the special character trigger `{selectall}`?

https://github.com/testing-library/user-event#special-characters

Hope that helps 😄

Reply:
> No such thing as a silly question to me, just an opportunity for knowledge sharing 😃
> @blestab are you referring to the special character trigger `{selectall}`?
> 
> https://github.com/testing-library/user-event#special-characters
> 
> Hope that helps 😄

Thanks @JacobMGEvans 😄, yep that's the one. I'll checkout the shared link